### DISCUSSION TITLE
Adição e salvamento

### DISCUSSION THREAD

**Discussion by ZillinNava** [YuriAlessandro/gerador-ficha-tormenta20#373] [Mapa Interativo de Arton]
Eu enquanto mexia no mapa interativo, acabei notando que não tem a opção de salvar um mapa que você acabou de adicionar novas localizações. Eu acho que seria bem útil essa funcionalidade, já que até hoje o meu computador está aberto com a guia especifica que acabei alterando.

Uma ideia bem legal também, seria dar para adicionar algumas objetos, como você faz quando cria mapas para rpg.

**Comment by YuriAlessandro** [comment]
Isso era um bug que eu não tinha me dado conta ainda. A partir de agora, novas localizações estarão salvas no seu navegador. Mesmo se você fechar a aba, vai voltar com as localizações.

Isso só vai sumir se você limpar os cookings e dados de navegação do navegador.
